Tools/TestWebKitAPI/Tests/WebKitCocoa/Copying.mm

```diff
@@ -34,8 +34,10 @@
     [a setMinimumFontSize:10];
     [a setJavaScriptEnabled:NO];
 #if PLATFORM(IOS_FAMILY)
+    [a setShouldPrintBackgrounds:NO];
     [a setJavaScriptCanOpenWindowsAutomatically:YES];
 #else
+    [a setShouldPrintBackgrounds:YES];
```

Comment:
Why are you setting different values on iOS / macOS?

(Same question further down)